# generated by patch-package 6.4.10
#
# command:
#   npx patch-package @milkdown/theme-nord
#
# declared package:
#   @milkdown/theme-nord: 5.5.0
#
diff --git a/node_modules/@milkdown/theme-nord/lib/index.es.js b/node_modules/@milkdown/theme-nord/lib/index.es.js
index 4b40df8..05332dc 100644
--- a/node_modules/@milkdown/theme-nord/lib/index.es.js
+++ b/node_modules/@milkdown/theme-nord/lib/index.es.js
@@ -283,11 +283,11 @@ const slots = () => ({
   icon: (id) => {
     const span = document.createElement("span");
     span.className = "icon material-icons material-icons-outlined";
-    span.textContent = iconMapping[id].icon;
+    span.textContent = iconMapping[id]?.icon || id;
     return span;
   },
   label: (id) => {
-    return iconMapping[id].label;
+    return iconMapping[id]?.label || id;
   }
 });
 const view = ({ css }) => css`
